envs/quadrotors/quad_multi_model.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, cfg, obs_space, timing, self_obs_dim=18, neighbor_obs_dim=6,
         self.neighbor_encoder_out_size = calc_num_elements(self.neighbor_encoder, (self.neighbor_obs_dim,))
 
         # Feed forward self obs and neighbor obs after concatenation
-        self.feed_forward = fc_layer(self.encoder_out_size + self.neighbor_encoder_out_size, cfg.hidden_size, spec_norm=self.use_spectral_norm)
+        self.feed_forward = fc_layer(self.self_encoder_out_size + self.neighbor_encoder_out_size, cfg.hidden_size, spec_norm=self.use_spectral_norm)
```

Comment:
I trust you that this works. Are we still defining the encoder_out_size? I think it is used in some other code

Reply:
We defined encoder_out_size in class EncoderBase, and initialize it to -1. However, we have never defined or update the value of encoder_out_size in this file (quad_multi_model.py). Therefore, we should definitely avoid to use encoder_out_size here, unless we set self.encoder_out_size =  self.self_encoder_out_size